packages/calypso-e2e/src/lib/test-account.ts

```diff
@@ -81,16 +87,41 @@ export class TestAccount {
 	/**
 	 * Retrieves the Time-based One-Time Password (a.k.a. verification code) from
 	 * the secret file if defined for the current account.
+	 *
+	 * @returns {string} Generated TOTP value.
+	 * @throws {Error} If TOTP secrets are missing for the user.
 	 */
-	getTOTP(): string | undefined {
+	getTOTP(): string {
```

Comment:
Throw an error instead of returning an `undefined`.

I personally do not think returning an `undefined` or `null` is useful in situations like this where the TOTP secret doesn't exist yet the code is being asked to generate one.